upload-api/email.js

```diff
@@ -1,3 +1,5 @@
+import fetch from '@web-std/fetch'
```

Comment:
wondering if we should just force node18? thoughts?

Reply:
I'm down! I don't know how big a change this is likely to be tho - will we need to use 18 for everything in the system or can we just change for one stack?

Reply:
I'm down! I don't know how big a change this is likely to be tho - will we need to use 18 for everything in the system or can we just change for one stack?

Reply:
for everything. it hopefully is just change version in https://github.com/web3-storage/w3infra/blob/main/stacks/index.js#L21 + CI (even in seed we are already using https://github.com/web3-storage/w3infra/blob/main/seed.yml)

Reply:
ah shoot - it looks like this will require updating SST. I'd like to avoid that side quest for now, so will stick with 16 I think.

Reply:
here's the error I was seeing, fwiw: 

<img width="800" alt="Screenshot 2023-06-20 at 3 12 39 PM" src="https://github.com/web3-storage/w3infra/assets/1113/552668de-9cea-4555-a507-1ed40a8d47a1">